### DISCUSSION TITLE
Seeking Guidance on Path Length settings

### DISCUSSION THREAD

**Discussion by Tbuttts21** [firemodels/fds#12086] [Announcements]
Hi all, 

Historically I have been using the following setting for my FDS runs: 

&RADI NUMBER_RADIATION_ANGLES=600, PATH_LENGTH=30.0 /

where the path length is equal to the distance between my target and the burner fire. 

My question is whether using the distance between the target and the fire base is a valid choice for the path length. I have seen some literature that suggests that this is a decent approximation for far-field targets. My secondary question is whether the path_length should change when wind is introduced. For example, if I add a 5 m/s wind in the direction of the target, the flame will be much closer to the target... should the path length change to account for this reduced distance between the flame and the target. 

Thanks!

**Comment by rmcdermo** [comment]
This is not a code issue.  Let's transfer this to a Discussion.  I will answer there.

**Comment by rmcdermo** [comment]
> My question is whether using the distance between the target and the fire base is a valid choice for the path length.

Yes, when there is water vapor and CO2 in the atmosphere, it is good to account for this when looking at radiative flux to a distant target.

> My secondary question is whether the path_length should change when wind is introduced.

This depends on the size of the fire.  If you really are in a point source regime, say 1 m flame and 30 m to target, then I would not think you need to change the path length.  If the distance to the target is cut in half by the wind and tilt of the flame, then I would factor that in.

**Reply by Tbuttts21** [reply]
Suppose the distance between the target and fire were cut in half due to the flame tilt, would it then be appropriate to divide the path length by half?

**Comment by mcgratta** [comment]
The PATH_LENGTH is characteristic length scale used to compute an effective gray-gas absorption coefficient. It is not the actual distance that is used when calculating the heat flux from A to B. I suggest you do a quick sensitivity analysis to see how much it matters.

**Reply by Tbuttts21** [reply]
Agreed. Can start doing sensitivity analyses from now on--I suppose my concern was with what value of Path Length I could deem as being "accurate". What I'm interested in is the heat flux on a distant object, so changing the path length around to get a desired heat flux isn't a viable option.

**Reply by drjfloyd** [reply]
The SFPE handbook has some plots of the expected atmospheric absorption as a function of the humidity pressure path length between the source and target.  With the default atmospheric CO2 and H2O, the default FDS pathlength will greatly underpredict the radiant flux. The attached is some calcs I did a while ago where the SFPE y indicates a percent humidty and the FDS x y is FDS pathlength percent humidty.  You could do something similar and figure out what pathlength is needed for a non-windblown fire. That pathlength would be too large in the wind blown case, but too large would mean reducing the atmospheric absorption which is at least conservative.  Could also zero out CO2 and H2O in the atmosphere.  That would add about a 20 % conservatism in the non-wind blown case and less with wind. 

![image](https://github.com/firemodels/fds/assets/12799217/36ccf29d-9753-4083-b476-91220900ed3d)

**Reply by Tbuttts21** [reply]
Thank you! This is very helpful... Out of curiosity, is there a specific code you can add to .fds files to record transmittance... I can't say that I've encountered that functionality yet. Or is this derived from equations in the technical guide. Thanks again!

**Reply by drjfloyd** [reply]
Transmittance is 1 - obscuration. Obscuration can be measured with a PATH OBSCURATION device. You can do the conversion either in post processing or using CTRL to do the math and outputting the results.

**Reply by Tbuttts21** [reply]
Makes sense. Thank you again.

**Reply by johodges** [reply]
We did some analysis on the impact of path length on the heat flux predictions in exterior tank-to-tank fire spread in a review for the Chemical Safety Board. The report was recently made public at the link below. There is some discussion in the hand calculations section (starting page A-12) as well as in the uncertainty section of the CFD modeling (starting page A-21):

https://www.csb.gov/assets/1/20/appendix_d_-_fire_protection_evaluation_report,_perspectives_on_tank_farm_fire,_itc_deer_park,_tx,_march_2019.pdf?16941

**Reply by Tbuttts21** [reply]
Thanks for sharing johodges. I think we will need to do some further analysis for the particular situations we're concerned with, but this is some great information to start from. Thanks again!
